components/back-button/index.js

```diff
@@ -0,0 +1,5 @@
+import { StyledBackButton } from "./back-button.styled";
+
+export default function BackButton() {
+  return <StyledBackButton href="/">back</StyledBackButton>;
+}
```

Comment:
Thinking of makeing the component more "reusabel".

What do you think of a prop that holds the information about, what to display?

```suggestion
export default function BackButton({label}) {
  return <StyledBackButton href="/">{label}</StyledBackButton>;
}
```

Instead of a hardcoded `back` ?

Reply:
good idea, thanks! fixed